Userland/Libraries/LibJS/Runtime/Intl/DurationFormat.cpp

```diff
@@ -29,6 +29,22 @@ DurationFormat::Style DurationFormat::style_from_string(StringView style)
     VERIFY_NOT_REACHED();
 }
 
+StringView DurationFormat::style_to_string(Style style)
```

Comment:
I didn't want to put them in the first since they're not actually used in the first, but they're also not really worthy of a separate commit, so I'll just put them in the first.